src/components/Drawer/MiniSideBar.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import { Button, Drawer, Typography, SvgIcon } from "@mui/material";
+import { PropsWithChildren } from "react";
+import CampaignOutlinedIcon from "@mui/icons-material/CampaignOutlined";
+import AccountBalanceOutlinedIcon from "@mui/icons-material/AccountBalanceOutlined";
+import AccountBoxOutlinedIcon from "@mui/icons-material/AccountBoxOutlined";
+import HeadsetMicOutlinedIcon from "@mui/icons-material/HeadsetMicOutlined";
+import InsertPhotoOutlinedIcon from "@mui/icons-material/InsertPhotoOutlined";
+import LightbulbOutlinedIcon from "@mui/icons-material/LightbulbOutlined";
+
+type RouteOption = {
+  label: string;
+  href: string;
+  icon: React.ReactNode;
+  disabled?: boolean;
+};
+
+const drawerWidth = 120;
+export default function MiniSideBar({ children }: PropsWithChildren) {
+  const dashboardRoutes: RouteOption[] = [
+    {
+      label: "Campaigns",
+      href: "/user/main/campaign",
+      icon: (
+        <CampaignOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    // Possible future enhancements, not visible to user but help keep spacing
+    {
+      label: "Creatives",
+      href: "/user/main/creatives",
+      icon: (
+        <LightbulbOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
```

Comment:
Let's not repeat specific colours like this. At very least use a constant, or even better configure this as one of the theme colours and use that. This looks like this is what we should use as `text.secondary`?